generator.rb

```diff
@@ -0,0 +1,51 @@
+def is_vowel?(letter)
+  return ["a", "e", "i", "o", "u"].include?(letter)
```

Comment:
You miss upper case characters here! one way to fix this is using `letter.downcase()`